v2/internal/frontend/desktop/windows/frontend.go

```diff
@@ -366,7 +366,14 @@ func (f *Frontend) setupChromium() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = settings.PutIsZoomControlEnabled(false)
+
+
+	if opts := f.frontendOptions.Windows; opts != nil && opts.ZoomFactor > 0.0 {
+		log.Print(fmt.Sprintf("Zoom: %d", opts.ZoomFactor))
```

Comment:
If the default is `1.0` it will print this each time. Did you mean to remove it after testing?

Reply:
oh, indeed, sorry :)

Will remove it

Reply:
removed